test/org/zaproxy/zap/utils/LocaleUtilsUnitTest.java

```diff
@@ -160,4 +162,30 @@ public void shouldNotMatchInvalidMessagesPropertiesFilesWithCreateMessagesProper
             assertThat(file, pattern.matcher(file).matches(), is(equalTo(false)));
         }
     }
+    
+    @Test
+    public void shoudAvailableLocalesBeNonEmpty() {
+        // Given
+        List<String> locals = LocaleUtils.getAvailableLocales();
+
+        // When
+        int localSize = locals.size();
+
+        // Then
+        assertTrue(localSize > 0);
+    }
+
+    @Test
+    public void shoudTheDefaultLocalBeEnglish() {
+        // Given
+        List<String> locals = LocaleUtils.getAvailableLocales();
+
+        // When
+        String defaultLocal = locals.get(0);
+
+        // Then
+        assertThat(defaultLocal, is(equalTo("en_GB")));
```

Comment:
Could you ensure locale has an "e" throughout, so that there's no confusion down the road?